ui/packages/catalog-ui-search/src/main/webapp/component/list-item/list-item.view.js

```diff
@@ -72,9 +77,6 @@ module.exports = Marionette.LayoutView.extend({
   setupListActions: function() {
     this.listActions.show(DropdownView.createSimpleDropdown({
       componentToShow: ListInteractionsView,
-      dropdownCompanionBehaviors: {
```

Comment:
❓ Did you mean to remove these?

Reply:
not sure why this is removed, i don't recall doing it on purpose, but some of these code changes go back months now. I'll put it back and see what happens